tests/ciscotest.rb

```diff
@@ -204,4 +204,17 @@ def get_interface_cleanup_config(intf_name)
       ["default interface #{intf_name}"]
     end
   end
+
+  def compatible_interface?
```

Comment:
Please change this method name to `mt_full_interface?`.  This is very specific to mtfull and now that it's not in a standalone file, we need to be more specific with the name.